default/scripting/buildings/INTERSPECIES_ACADEMY.focs.txt

```diff
@@ -0,0 +1,94 @@
+            Species name = RootCandidate.Species
+            OwnedBy empire = RootCandidate.Owner
+        ]
+        Number low = 0 high = 0 condition = Described description = "Academies of this species already exist in the empire" condition = And [
+            Planet
+            Enqueued type = Building name = "BLD_INTERSPECIES_ACADEMY"
+            Species name = RootCandidate.Species
+            OwnedBy empire = RootCandidate.Owner
+            Not Object id = RootCandidate.PlanetID
+        ]
+        Number low = 0 high = 6 condition = And [
+            Building name = "BLD_INTERSPECIES_ACADEMY"
+            OwnedBy empire = Source.Owner
+        ]
+        
+    ]
+    EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
+    effectsgroups = [
+        EffectsGroup 
+            scope = NumberOf number = 1 condition = And [
+                Building name = "BLD_INTERSPECIES_ACADEMY"
+                OwnedBy empire = Source.Owner
+                Not Object id = Source.ID
+                Species name = Source.Planet.Species
+            ]
+            activation = And [
+                Source
+                Random probability = 0.3
+            ]
+            effects = [
+                GenerateSitRepMessage
+                    message = "Destroying fake duplicate design academy %planet%."
+                    label = "EFFECT_BLACKHOLE_LABEL"
+                    NoStringtableLookup
+                    icon = "icons/building/blackhole.png"
+                    parameters = [
+                        tag = "planet" data = Target.PlanetID
+                    ]
+                    empire = Source.Owner
+                Destroy
+            ]
+            
+        EffectsGroup
+            scope = And [
+                Object id = Source.PlanetID
+                Planet
+                OwnedBy empire = Source.Owner
+                Species
+            ]
+            activation = Source
+            priority = [[LATE_PRIORITY]]
+            effects = [
+                GenerateSitRepMessage
+                    message = "Additional effect by the %species% on %planet% joining the design academy. Each design academy adds 0.05 to %rawtext%"
+                    label = "EFFECT_BLACKHOLE_LABEL"
+                    NoStringtableLookup
+                    icon = "icons/building/blackhole.png"
+                    parameters = [
+                        tag = "planet" data = Source.PlanetID
+                        tag = "species" data = Source.Planet.Species
+                        tag = "rawtext" data = EmpireMeterValue empire = Source.Owner meter = "METER_IMPERIAL_PP_TRANSFER_EFFICIENCY"
+                    ]
+                    empire = Source.Owner
+                SetEmpireMeter empire = Source.Owner meter = "METER_IMPERIAL_PP_TRANSFER_EFFICIENCY" value = Value + 0.05
+                SetEmpireMeter empire = Source.Owner meter = "METER_IMPERIAL_PP_USE_LIMIT"           value = Value + 1
```

Comment:
needs limits to ensure efficiency doesn't go over 1

Reply:
Also I cap the efficiency at 1.0f in the backend see commit a5faf12433798f4faa945cd5608529da39380867 .